code/style.css

```diff
@@ -1,4 +1,322 @@
 html {
-  background: #0025ff;
+  background: #fff;
+}
+
+.container {
+  max-width: 1440px;
+  margin: 0 auto;
+  padding: var(--grid-outer-padding);
+}
+
+body {
+  font-family: 'Nunito', sans-serif;
+}
+
+h2 {
+  font-size: 2rem;
+}
+
+em {
+  font-family: 'Playfair Display', serif;
+}
+
+/* header */
+
+.mobile-only-header {
+  display: flex;
+  flex-direction: column;
+  text-align: center; 
+  padding: 10px;
+  height: 64px;
+}
+
+.desktop-only-header,
+.tablet-only-header {
+  display: none;
+}
+
+.logo {
+  height: 75px;
+  border-radius: 25px;
+}
+
+nav {
+  display:flex;
+  justify-content:center;
+  font-size: 1.3em;
+}
+
+nav p {
+  margin: 0px;
+  padding: 0px 0px 0px 20px;
+}
+
+a, a > span {
+  position: relative;
+  color: inherit;
+  text-decoration: none;
+}
+a:before, a:after, a > span:before, a > span:after {
+  content: '';
+  position: absolute;
+  transition: transform .5s ease;
+}
+
+.hover:before {
```

Comment:
Here! hover:hover - it took me some time to understand what it was reffering to.

Reply:
Yes I can see that. I think I could actually replace this with a:hover - that would make more sense 😂